main.go

```diff
@@ -51,84 +52,89 @@ var validDBNameRegex = regexp.MustCompile(`^[_\-a-zA-Z0-9]+$`)
 var validSchemaNameRegex = regexp.MustCompile(`^[_\-a-zA-Z0-9]+$`)
 
 type dbCon struct {
-	driver   string
-	host     string
-	port     string
-	dbname   string
-	user     string
-	password string
-	sslmode  string
+	Driver   string `json:"driver"`
+	Host     string `json:"host"`
+	Port     string `json:"port"`
+	Name     string `json:"name"`
+	Username string `json:"username"`
+	Password string `json:"password"`
+	ModTrack string `json:"modtrack"`
+	Alias    string `json:"alias,omitempty"`
+	SSLMode  string `json:"sslmode,omitempty"`
 }
 
 func (c *dbCon) makeConStr() string {
-	s := fmt.Sprintf("host=%v dbname=%v user=%v", c.host, c.dbname, c.user)
-	if c.port != "0" && c.port != "" {
-		s += fmt.Sprintf(" port=%v", c.port)
+	s := fmt.Sprintf("host=%v dbname=%v user=%v", c.Host, c.Name, c.Username)
+	if c.Port != "0" && c.Port != "" {
+		s += fmt.Sprintf(" port=%v", c.Port)
 	}
-	if c.password != "" {
-		s += fmt.Sprintf(" password=%v", c.password)
+	if c.Password != "" {
+		s += fmt.Sprintf(" password=%v", c.Password)
 	}
-	if c.sslmode != "" {
-		s += fmt.Sprintf(" sslmode=%v", c.sslmode)
+	if c.SSLMode != "" && c.SSLMode != "allow" { // allow is not an accepted value for sslmode
```

Comment:
Could you elaborate why "allow" would not be acceptable?

Reply:
This is what the driver told me. There does not seem to be an alternative.
```
pq: unsupported sslmode "allow"; only "require" (default), "verify-full", "verify-ca", and "disable" supported
```